knowledge-base/hide-table-borders-extending-to-the-next-page.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Hide table borders extending to the next page
+description: How to hide table borders that extend to the end of the page
+type: how-to
+page_title: Remove extra table borders when the table is paged
+slug: hide-table-borders-extending-to-the-next-page
+position: 
+tags: 
+ticketid: 1453320
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+By design, the borders of the [Table item](https://docs.telerik.com/reporting/table-working-with-table-cross-table-list-items) will be drawn to the end of the page to indicate that the table is continued to the next page.
```

Comment:
you may use ../table-working-with-table-cross-table-list-items instead of the full path

Reply:
path to our documentation and KBs should be relative